README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
Good! 
Be sure to return a value in your render methods. eslint: react/require-render-return

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Inelegant, Not reusable